sample_apps/java/src/main/java/com/amazonaws/services/timestream/CrudAndSimpleIngestionExample.java

```diff
@@ -207,6 +209,13 @@ public void writeRecords() {
         try {
             WriteRecordsResult writeRecordsResult = amazonTimestreamWrite.writeRecords(writeRecordsRequest);
             System.out.println("WriteRecords Status: " + writeRecordsResult.getSdkHttpMetadata().getHttpStatusCode());
+        } catch (RejectedRecordsException e) {
+            System.out.println("RejectedRecords: " + e);
+            for (RejectedRecord rejectedRecord : e.getRejectedRecords()) {
+                System.out.println("Rejected Index " + rejectedRecord.getRecordIndex() + ": "
+                        + rejectedRecord.getReason());
+            }
+            System.out.println("Other records have already been wrote successfully. ");
```

Comment:
Can you update "wrote" to "written"? Or maybe simplify the phrase to "Other records were written successfully". Same for other sample apps.